index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device=width">
+        <title>Weather Report</title>
+        <link rel="preconnect" href="https://fonts.gstatic.com">
+        <link href="https://fonts.googleapis.com/css2?family=Raleway&display=swap" rel="stylesheet">    
+        <link href="styles/index.css" rel="stylesheet" type="text/css"/>
+    </head>
+
+    <body>
+        <header class= "header__header">
+            <h1>Weather Report</h1>
+            <span> For the city of
+                <span id="headerCityName" class="header__city-name"></span></span>
+        </header>
+            <section class="temperature__section">
+                <h2>Temperature:</h2>
+            <div class="temperature__content">
+                <div class="temperature__controls">
+                    <button id="increaseTempControl">warmer</button>
+                    <button id="decreaseTempControl">cooler</button>
+                </div>
+                <span id="tempValue"></span>
+                </div>
```

Comment:
oops try not to use `div` elements! stick with semantic html tags, like `section`, `main`, etc.